release-notes.md

```diff
@@ -8,6 +8,18 @@ description: Release notes and change logs for the Adobe Experience Platform Mob
 
 Please take a moment to fill out a [short survey](https://www.surveymonkey.com/r/AEPDocs) on how we can better assist you with enabling Adobe Experience Cloud solutions and services on your mobile apps.
 
+## January 27, 2020
+
+The following updates were made in this release:
+
+**iOS Core 2.4.0**
+
+* Added a new property `eventUniqueIdentifier` to the `ACPExtensionEvent` class. 
```

Comment:
- Please add a comma between "property" and the property name and then between the property name and "to", like this:
"...a new property, `eventUniqueIdentifier`, to..."